frontend/src/high-dimensional/HighDimensional.vue

```diff
@@ -33,18 +34,27 @@ export default {
   name: 'HighDimensional',
   data() {
     return {
+      runsArray: [],
       config: {
         searchText: '',
         displayWordLabel: true,
         dimension: '2',
         reduction: 'tsne',
+        showingRun: '',
```

Comment:
Can we have a better name? 'showingRun' is quite confusing 😢

Reply:
Of course, any suggestion?